lib/impl/AwsS3Tvm.js

```diff
@@ -109,9 +111,47 @@ class AwsS3Tvm extends Tvm {
       secretAccessKey: stsRes.Credentials.SecretAccessKey,
       sessionToken: stsRes.Credentials.SessionToken,
       expiration: stsRes.Credentials.Expiration,
-      params: { Bucket: params.s3Bucket }
+      params: { Bucket: bucketName }
     }
   }
 }
 
+function checkAndCreateS3Bucket(namespace, prefix, region) {
+  return new Promise((resolve, reject) => {
+    var s3 = new AWS.S3({region: region});
+    const bucketName = getBucketName(namespace, prefix)
+    var params = {
+    Bucket: bucketName
+    };
+    //check if bucket exists
+    s3.headBucket(params, function(err, data) {
+     if (err){
+       if(err.statusCode == 404 ) {
+         console.log("Trying to create bucket - " + bucketName)
```

Comment:
instead of console.log please use the aio logger:
```
const logger = require('@adobe/aio-lib-core-logging')('@adobe/aio-tvm')
logger.info('my log')
```